mantidproject/mantid
# Remove FunctionBrowser in ALFView

**Description of work.**
This PR removes the FunctionBrowser seen in the ALFView interface and replaces it with a QLineEdit which displays the PeakCentre, and a Fit Status label.

**To test:**
1. Open `Direct`->`ALFView` interface
1) Load data [ALF82301](file:///home/mlc47243/Documents/other-documents/ALFView/ALF82301.zip)
2) Click on `Pick` tab
5) Expand `Rebin` section. Filter by d-spacing range (optional) `5.5,0.01,6`
6) Expand `Plotting detector spectra` section
6) Click on `Select whole tube` icon
9) Right click on peak of interest and select `Extract Single Tube`
11) Press `Update Estimate` button. The Peak Centre box should be updated from zero
12) Press `Fit` button. The peak centre box will be updated again, and a Fit status displayed below this box.
13) Change the Peak Centre to be outside the Fit Range, and then try `Fit` or `Update Estimate`. There should be a warning message displayed.


Part of #34604

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
